backend/alert/serializers.py

```diff
@@ -43,34 +44,10 @@ class RegistrationSerializer(serializers.ModelSerializer):
         help_text="The current status of the watched section. Options and meanings: "
         + string_dict_to_html(dict(StatusUpdate.STATUS_CHOICES)),
     )
-    is_active = serializers.SerializerMethodField(
-        read_only=True,
-        help_text=dedent(
-            """
-        True if the registration would send an alert hen the watched section changes to open,
-        False otherwise. This is equivalent to not(notification_sent or deleted or cancelled).
-        """
-        ),
-    )
-    is_waiting_for_close = serializers.SerializerMethodField(
-        read_only=True,
-        help_text=dedent(
-            """
-        True if the registration is waiting to send a close notification to the user
-        once the section closes.  False otherwise.
-        """
-        ),
-    )
```

Comment:
We no longer need these SerializerMethodFields since the auto-docs should infer help text from property docstring now, pending the documentation-update branch being merged.